### PR TITLE
Convolute

### PR DESCRIPTION
**Pull request by PopeJuliusII**
I created a function to return the requisite total from three inputs.

### CODE REVIEW

File: convolute.py [hunk 1]
```diff
@@ -0,0 +1,2 @@
+def triple_convolute(A, B, C):
```
Comment (adleonis) [inline convolute.py]: comments.  what is this function for

Comment (adleonis) [reply convolute.py]: also define for other users what the output data format is...

Comment (adleonis) [inline convolute.py]: funtion name is wrong

Comment (adleonis) [inline convolute.py]: call the function convolute()

**Review by adleonis — CHANGES_REQUESTED** [review]

**Comment by PopeJuliusII** [comment]
I added the docstring and renamed as requested.

**Review by adleonis — APPROVED** [review]
Well done!

**Pull request merged**